Feat(Developer guide): CI/CD introduction (#137)
kestra-io/kestra.io: docs/developer-guide/cicd/README.md

# Create a CI/CD pipeline for Kestra

## Introduction

> In Kestra, the number of flows you developed can grow quickly, and maintaining them can be a challenge.

Reviewer: ```suggestion
In Kestra, the number of flows you develop can grow quickly, and maintaining them can get challenging.	
```

> CI/CD pipelines are a great way to automate the validation and the deployment of your flows.

Reviewer: ```suggestion
CI/CD pipelines are a great way to automate your flows' validation and deployment.
```

> This section will guide you through the steps to create a CI/CD pipeline.

Reviewer: ```suggestion
This section will guide you through creating a CI/CD pipeline for your Kestra flows.
```

## Why a CI/CD pipeline?

> CI/CD pipelines provide an easy and reliable way to ensure that your flows are always deployed as soon they're done.

Reviewer: As soon as they are done with? 👀

Reviewer 2: > are always deployed as soon they're in your source control.

That's the point of a continuous integration right ? You commit, it trigger a workflow

Reviewer: `they are in your source control.`

Yeah, that makes sense. I wasn't sure if there was more to it. Thanks :)

> It integrates well in VCS (Version Control System) like Git, and can be easily integrated in your development workflow.

Reviewer: ```suggestion
It integrates nicely with VCS (Version Control System) like Git and can easily integrate into your development workflow.
```
